event/event.go

```diff
@@ -0,0 +1,11 @@
+package event
+
+type Event interface {
+	GetEventType() string
+
+	GetEventAttributeKey() string
+
+	GetEventAttributeValue() string
+
+	EventHandler() error
+}
```

Comment:
What if we could add a `ResultEvent` to the handler to use?

```suggestion
package event

import ctypes "github.com/tendermint/tendermint/rpc/core/types"

type Event interface {
	GetEventType() string

	GetEventAttributeKey() string

	GetEventAttributeValue() string

	EventHandler(ctypes.ResultEvent) error
}
```

Reply:
Yes, I agree.
So we can query or verify with `ResultEvent`.

Reply:
Fixed in [133371d](https://github.com/medibloc/panacea-doracle/pull/15/commits/133371d7ed9d36e9098f466b94346c31aae260ce).